### ISSUE TITLE
Session [(null externalkey)] not available and is not among the last 1000 terminated sessions. Active sessions are[ext. key xa154aa5-4bef-41c1-a0b8-610c462493a6]

### ISSUE THREAD

**Issue by ghost** [facebook/php-webdriver#578]
**The problem**

The tests worked successfully before i update of the selenium plugin for jenkins. The tests run locally in the selenium-standalone. But unfortunately not in the selenium grid.

```php

public function setUp() : void {
  $desiredCapabilities = DesiredCapabilities::firefox();
  $this->Driver = RemoteWebDriver::create( 'http://JENKINS_URL:4444/wd/hub', $desiredCapabilities );
}

public function tearDown() : void {
  if ( $this->Driver ) {
    $this->Driver->quit();
  }
}

```

### Details

* Php-webdriver version:  1.6.0
* PHP version: 7.1
* Selenium server version: 3.7.1
* Operating system: Linux
* Browser used + version: 58.0.1

**Comment by williamdes** [comment]
Using manual api calls works fine.

```bash
curl -X POST -d '{"desiredCapabilities":{"browserName":"chrome"}}' http://localhost:4444/wd/hub/session
{
  "status": 0,
  "sessionId": "de94ada5b64230734a7620ebf3914b24",
  "value": {
    "acceptInsecureCerts": false,
    "acceptSslCerts": false,
    "applicationCacheEnabled": false,
    "browserConnectionEnabled": false,
    "browserName": "chrome",
    "chrome": {
      "chromedriverVersion": "2.41.578700 (2f1ed5f9343c13f73144538f15c00b370eda6706)",
      "userDataDir": "\u002ftmp\u002f.org.chromium.Chromium.7tiGlQ"
    },
    "cssSelectorsEnabled": true,
    "databaseEnabled": false,
    "goog:chromeOptions": {
      "debuggerAddress": "localhost:40565"
    },
    "handlesAlerts": true,
    "hasTouchScreen": false,
    "javascriptEnabled": true,
    "locationContextEnabled": true,
    "mobileEmulationEnabled": false,
    "nativeEvents": true,
    "networkConnectionEnabled": false,
    "pageLoadStrategy": "normal",
    "platform": "Linux",
    "rotatable": false,
    "setWindowRect": true,
    "takesHeapSnapshot": true,
    "takesScreenshot": true,
    "unexpectedAlertBehaviour": "",
    "version": "69.0.3497.81",
    "webStorageEnabled": true,
    "webdriver.remote.sessionid": "de94ada5b64230734a7620ebf3914b24"
  }
}
 curl -X POST -d '{"desiredCapabilities":{"browserName":"firefox"}}' http://localhost:4444/wd/hub/session
{
  "value": {
    "sessionId": "1314b8ff-5451-4938-88eb-8882d9fdc6fb",
    "capabilities": {
      "acceptInsecureCerts": false,
      "browserName": "firefox",
      "browserVersion": "62.0",
      "moz:accessibilityChecks": false,
      "moz:headless": false,
      "moz:processID": 24522,
      "moz:profile": "\u002ftmp\u002frust_mozprofile.v67ufRNvIgwJ",
      "moz:useNonSpecCompliantPointerOrigin": false,
      "moz:webdriverClick": true,
      "pageLoadStrategy": "normal",
      "platformName": "linux",
      "platformVersion": "4.15.0-32-generic",
      "rotatable": false,
      "timeouts": {
        "implicit": 0,
        "pageLoad": 300000,
        "script": 30000
      },
      "webdriver.remote.sessionid": "1314b8ff-5451-4938-88eb-8882d9fdc6fb"
    }
  }
curl -X POST -d '{"url":"https://google.fr"}' http://localhost:4444/wd/hub/session/1314b8ff-5451-4938-88eb-8882d9fdc6fb/url
{"value": null}
curl -X GET http://localhost:4444/wd/hub/session/1314b8ff-5451-4938-88eb-8882d9fdc6fb/url
{"value":"https://google.fr"}



```

**Comment by williamdes** [comment]
Fixed by PR #560 
see : https://github.com/facebook/php-webdriver/blob/be980edb03742db1774e33d79fa6507280d4b505/lib/Remote/HttpCommandExecutor.php

**Comment by cpintea** [comment]
I get this error when i run tests on MAC/safari all the time. I am using selenium grid. Is this behavior on MAC/Safari generated by the same issue that cause the Firefox behavior?. The safari browser is opened but the next command send to the safari driver fail with the  exception "not available and is not among the last 1000 terminated sessions". The "PR #560" will fix also this issue?
